GothicVRProject/GothicVR
# fix(WorldMeshChunking): Whenever there are no lights, the algorithm p…

…roduces all meshes (in smallest chunks) again.

# To test
1. Disable all on Feature Flags except "SkipMainMenu" and "LoadWorldMesh"
2. Check if the world is completely rendered (issue: https://github.com/GothicVRProject/GothicVR/issues/343)
